testsuite/pom.xml

```diff
@@ -408,6 +408,22 @@
                    <artifactId>maven-install-plugin</artifactId>
                    <executions><execution><id>default-install</id><phase>none</phase></execution></executions>
                </plugin>
+
+               <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-pmd-plugin</artifactId>
+                <version>3.5</version>
+                <configuration>
+                    <rulesets>
+                        <ruleset>https://raw.githubusercontent.com/wildfly/wildfly/master/testsuite/shared/src/main/config/pmd.xml</ruleset>
+                    </rulesets>
```

Comment:
@ctomc I've been trying to set a local path, but the issue is that depending where you invoke the maven build, the local path change. Trying to make it depends on ${basedir} did not work either. I ended up giving up and simply put the URL as a workaround.

Do you know a way to set this path as being /path/to/parent/home/testsuite/shared/.../pmd.xml ?